ci-operator/step-registry/ipi/deprovision/azure/resourcegroup/ipi-deprovision-azure-resourcegroup-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# TODO: move to image
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+existing_rg=$(/tmp/yq r "${CONFIG}" 'platform.azure.resourceGroupName')
+
+# az should already be there
+command -v az
+
+# set the parameters we'll need as env vars
+AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+AZURE_AUTH_CLIENT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientId)"
+AZURE_AUTH_CLIENT_SECRET="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientSecret)"
+AZURE_AUTH_TENANT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .tenantId)"
+
+# log in with az
+az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}"
```

Comment:
What are your thoughts on using `--output none` in the login command? There is nothing terribly sensitive in the login output, but it is also unnecessary account details to include in the CI artifacts. I'm a bit torn on this.

Reply:
I think it's a good idea, and would enable it to improve security. We would still see error messages.

However, it'd be good to have a standardized approach for this across the jobs/repo.